.circleci/config.yml

```diff
@@ -443,6 +443,28 @@ jobs:
             echo "export DOCKER_BUILDKIT=" > $BASH_ENV
             build aztec-sandbox false arm64
 
+  aztec-p2p-bootstrap:
+    machine:
+      image: ubuntu-2204:2023.07.2
+    resource_class: large
+    steps:
+      - *checkout
+      - *setup_env
```

Comment:
This should now be built by yarn-project I think. You would be able to grab the build from there

Reply:
Doesn't yarn-project just produce a single image `yarn-project` with all of the built code? And if we want an actual deployable package image we need to have a Dockerfile which extracts the relevant built code for that package?